geospaas_harvesting/ingesters.py

```diff
@@ -501,7 +501,7 @@ def get_download_url(dataset_info):
 
 class URLNameIngester(MetanormIngester):
     """ Ingester class using FTP to read meta-data from the name of the dataset """
-    LOGGER = logging.getLogger(__name__ + '.FTPIngester')
```

Comment:
IF this ingestor works for both FTP and local files, then the docstring of it must be corrected in above line.

Reply:
sure